packages/css-library/src/stylesheets/modules/m-alert.scss

```diff
@@ -250,7 +253,7 @@ p.usa-alert-heading {
   &-information {
     border-top: scale-rem(1rem) solid $color-primary-alt-dark;
     .usa-alert:before {
-      content: "\F05A";
+      // content: "\F05A";
```

Comment:
I could not find any usages of the classes the icon is for `.usa-alert-full-width-information` in either vets-website or content-build so I'm just commenting this out.